src/features/slackapi/slackapi-utils.ts

```diff
@@ -21,10 +21,13 @@ namespace SlackApiUtilities {
   export const getSlackUsers = async (): Promise<Member[]> => {
     try {
       const result = await client.users.list();
-      return result.members;
+      if (result.members){
+        return result.members;
+      }
+      throw new Error(`Error while loading slack users list, ${result.error}`);
```

Comment:
This if statement can be one-liner.
```suggestion
      if (result.members.length) return result.members;

      throw new Error(`Error while loading slack users list, ${result.error}`);
```

Reply:
If statement has been fixed, but length won't be able to be checked, due to mocks not including the members field.

Reply:
restructured but still not checking for members.length as the error response from the API will not contain the members field. Would be great to know if there is a preferred way to error handle in this situation.